src/main/kotlin/com/sourcegraph/config/ConfigUtil.kt

```diff
@@ -141,11 +141,13 @@ object ConfigUtil {
 
     return try {
       val text = customConfigContent ?: getSettingsFile(project).readText()
-      val config = ConfigFactory.parseString(text).resolve()
+      var config = ConfigFactory.parseString(text).resolve()
       additionalProperties.forEach { (key, value) ->
-        config.withValue(key, ConfigValueFactory.fromAnyRef(value))
```

Comment:
There was a bug 😄 conigs are immutable

Reply:
Basically, we ignored "foldingRanges": "indentation-based" - how important for us is this option?